app/models/dfe_sign_in_user.rb

```diff
@@ -0,0 +1,25 @@
+module DfESignInUser
+  extend ActiveSupport::Concern
+
+  def self.begin_session!(session, dfe_sign_in_session)
+    session['dfe_user'] = {
```

Comment:
I think this implies the user is from the DfE itself — maybe `dfe_sign_in_user`?